### PR TITLE
Popup-Window: Setup

### PR DESCRIPTION
**Pull request by MoyasiGinko**
### In this milestone, I:
- Added `modal` section in html file
- Added modal properties in the `style.css` file
- Add modal funtion in `script.js` file
- Modified `style.css` file

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -18,4 +18,175 @@ document.documentElement.addEventListener('click', (e) => {
...
+//         <div>
+//           <button class="btn">See Project</button>
+//         </div>
+//       </div>
+//     </div>
+//   `;
+// }
+
+// for (let i = 0; i < projectData.length; i += 1) {
+//   // displayCard(i);
```
Comment (rotimiazeez) [inline script.js:97]: - Kindly ensure all commented codes to make your codebase more readable 🤝🏽

File: style.css [hunk 2]
```diff
@@ -759,6 +855,35 @@ header {
...
     padding: 12px 16px;
   }
+
+  /* Modal */
+  .popup-description {
+    display: flex;
+    justify-content: center;
+  }
+
+  .popup-description .card__skills {
```
Comment (rotimiazeez) [inline style.css:869]: Please ensure your desktop modal design buttons matches that of the original Figma design 🙏.
Your design
Screenshot_20230419_214132
Figma design
Screenshot_20230419_214411

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM👍🏽,
## Good job so far!
### To Highlight 👇🏽
- Good teamwork 👏🏾
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: script.js (hunk 1 above)
Comment (MoyasiGinko) [reply script.js:97]: @rotimiazeez Please, suggest if I have to delete the comments or should I keep them?

**Review by Strangeal — APPROVED** [review]
Hi @MoyasiGinko and @Gardimy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**